### DISCUSSION TITLE
I'm hacked could someone help

### DISCUSSION THREAD

**Discussion by gary2see** [community/community#138990] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm hacked could I get some help please

**Comment by queenofcorgis** [comment, marked as answer]
Hi @gary2see , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue. The best place to go for answers to **GitHub** account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

For any other accounts or devices, GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
